docroot/themes/contrib/civictheme/civictheme_library/components/00-base/_variables.base.scss

```diff
@@ -340,6 +340,12 @@ $civictheme-icon-sizes-default: (
   )
 );
 
+//
+// Icon symbols.
+//
+$civictheme-icon-symbols: null !default;
+$civictheme-icon-symbols-default: account, approve, bars, bullet, calendar-1, calendar-2, call, cancel, checked, close-outline, close, down-arrow-1, down-arrow-2, down-arrow-large, download-file, download, exclamation-mark-1, exclamation-mark-2, exclamation-mark-3, eye, facebook, help, image, information-mark, left-arrow-outline, left-arrow, linkedin, magnifier, map, megaphone, mortarboard, newspaper, pdf-file, progress, right-arrow-1, right-arrow-2, select-box, share, square, to-do, twitter, up-arrow-large, up-arrow, upper-arrow, upper-right-arrow, watch, word-file;
```

Comment:
please convert this to map in the same way as colours and fonts are mapped

Reply:
Converted to  to map.